index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+
+		<title>ToDo List</title>
+		<meta name="description" content="ToDo application" />
+		<meta name="author" content="Bartlomiej Zdybel" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+		<link rel="stylesheet" type="text/css" href="./index.css" />
+
+		<link rel="icon" href="icon_code.jpg" />
+	</head>
+
+	<body>
+		<div class="container">
+			<div class="task-list__container">
+				<form class="task-form task-form__add">
+					<h1 class="task-form__title">task list</h1>
+
+					<div class="task-form__input-wrapper">
+						<label class="task-form__label task-form__label--title" for="task">What must you do?</label>
+						<input
+							id="task"
+							class="task-form__input task-form__input--task-content"
+							placeholder="eg. Go to the cinema"
+						/>
+					</div>
+
+					<div class="task-form__input-wrapper">
+						<label class="task-form__label task-form__label--title" for="date">Enter the deadline</label>
+						<input
+							id="date"
+							type="date"
+							class="task-form__input task-form__input--date"
+							value="2019-03-12"
+						/>
+					</div>
+					<div class="task-form__button-wrapper">
+						<button class="task-form__button" type="submit">Add to list</button>
+					</div>
+				</form>
+
+				<ul class="task-list">
+					<li class="task-list__item">
+						<button class="task-list-item__button task-list-item__button--is-done" type="button">
```

Comment:
Is there any reason why you decided to use a button here? Does it lead to any action or is it just for styling purposes? The question is actually about "delete" button in "DELETED" list too.

Reply:
In the "DELETED" section this button has only a styling purposes, and i' ve chosen button because in my mind it was the most natural choice in this case.

Reply:
Every button should lead to some kind of interaction, this is not the case here. An icon with the exact same appearance woule be a good choice, so change it, please :)